src/grb_model.jl

```diff
@@ -104,7 +106,7 @@ end
 
 # read / write file
 
-function read_model(model::Model, filename::ASCIIString)
+function read_model(model::Model, filename::String)
```

Comment:
I have no motivation to spend time figuring this out and testing on different platforms. Should I add an ``isascii`` assert with a TODO?